assets/style.css

```diff
@@ -0,0 +1,1017 @@
+  text-decoration: none;
+}
+
+.link-c-red a:hover,
+.link-c-red:hover {
+  color: #f72210;
+}
+
+.link-c-red2 a:hover,
+.link-c-red2:hover {
+  color: #f92859;
+}
+
+.link-c-black a:hover,
+.link-c-black:hover {
+  color: black !important;
+}
+
+.link-u-red a:hover,
+.link-u-red:hover {
+  text-decoration: underline red !important;
+}
+
+.link-b-gray a:hover,
+.link-b-gray:hover {
+  background-color: #dbdcdd;
+}
+
+/* main */
+
+main {
+  max-width: 1310px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 0;
+}
+
+@media (min-width: 992px) {
+  .bg-lg-white {
+    background-color: white !important;
+  }
+}
+
+main .container-lg-fluid {
+  width: 100%;
+  padding-right: 15px !important;
+  padding-left: 15px !important;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+}
+
+@media (min-width: 992px) {
+  header {
+    position: static;
+  }
+}
+
+/* top navbar */
+
+.logo > img {
+  width: 132px;
+  margin-left: 15px;
+}
+
+.temp-icon > img {
+  width: 24px;
+  height: 24px;
+}
+
+.subscribe-btn,
+.login-btn {
+  padding: 6px 2px;
+  font-weight: 700;
+  font-size: 14px;
+  margin-left: 10px;
+  line-height: 14px;
+}
+
+.toggler-icon {
+  width: 50px;
+  height: 50px;
+  padding-left: 11px;
+  padding-right: 11px;
+}
+
+.toggler-icon > span,
+.toggler-icon::before,
+.toggler-icon::after {
+  display: block;
+  width: 24px;
+  height: 4px;
+  background-color: #fb9a94;
+  margin: 2px;
+}
+
+.toggler-icon::before,
+.toggler-icon::after {
+  content: "";
+}
+
+@media (min-width: 481px) {
+  .subscribe-btn {
+    flex-basis: 121px;
+    margin-right: 6px;
+  }
+
+  .subscribe-btn,
+  .login-btn {
+    padding: 6px 8px 6px 9px;
+    font-weight: 700;
+    font-size: 18px;
+    margin-left: 10px;
+    line-height: 14px;
+  }
+
+  .login-btn {
+    padding-right: 2px;
+  }
+
+  .subscribe-btn::after {
+    width: 8px;
+    height: 14px;
+    content: "";
+    display: inline-block;
+    margin-left: 10px;
+    background: url(images/icon-arrow-right-red.svg) center / contain no-repeat;
+  }
+
+  .toggler-icon > span,
+  .toggler-icon::before,
+  .toggler-icon::after {
+    width: 30px;
+  }
+
+  .toggler-icon {
+    width: 74px;
+  }
+}
+
+@media (min-width: 576px) {
+  .temp-icon {
+    margin-left: 23px;
+    margin-right: 5px;
+  }
+
+  .temp-deg {
+    font-size: 15px;
+  }
+}
+
+@media (min-width: 641px) {
+  .temp-location {
+    display: block !important;
+    padding-left: 8px;
+    font-size: 15px;
+  }
+
+  .temp-location > .state {
+    margin-right: 7px;
+  }
+}
+
+@media (min-width: 992px) {
+  .temp,
+  .login {
+    width: 210px;
+  }
+
+  .logo > img {
+    width: 396px;
+  }
+
+  .temp-date,
+  .temp-icon {
+    margin-left: 20px;
+  }
+
+  .temp-date {
+    margin-top: 10px;
+  }
+
+  .nav-top {
+    padding-top: 20px;
+    padding-bottom: 15px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .temp-date,
+  .temp-icon {
+    margin-left: 30px;
+  }
+}
```

Comment:
Here (and for most of your media queries) is a great opportunity to rely on bootstrap's [responsive breakpoints](https://getbootstrap.com/docs/4.5/layout/overview/#responsive-breakpoints).

Reply:
Again, I did my best on this one but still, I have the next comments: 

1.- Like in my previous comment, there are no defined bootstrap classes for w-md-100, w-lg-100, for example. 

2.- In some cases, it's more accurate to use pixels instead of the `m-` or `p-` bootstrap classes because there's no equivalence. 

3.- The project uses additional break points such as `201px` , `481px` and `641px` for the navbar. 

4.- Regarding the grid layout, I had a lot of discussion with my team to try really hard defining the first three-column layout with the pure bootstrap grid to no avail.  Then I decided to use grid to get it to work.  The problem wasn't actually the layout but the reordering of sections responsively just like in the original. For the rest, I used the bootstrap grids.